src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,19 @@
+.catalog {
+  display: flex;
+  justify-content: center;
+  flex-wrap: wrap;
+
+  &__card {
+    min-width: 198px;
```

Comment:
It's better to use fixed width for card instead of min-width to match task requirements.

Reply:
+